src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -365,6 +368,7 @@ DropDownGroup.propTypes = {
   placeholder: PropTypes.string,
   variant: PropTypes.oneOf(Object.values(LAYOUT_VARIANTS)),
   label: PropTypes.string,
+  hiddenLabel: PropTypes.bool,
```

Comment:
Please describe the use of **hiddenLabel**  new attribute so the integrator will know the use of **hiddenLabel** .

Reply:
Updated in props. and also in md file